### ISSUE TITLE
[DRAGONAPI] Galacticraft oxygen gear/mask/tanks are not visible with Reika mods.

### ISSUE THREAD

**Issue by MJRLegends** [micdoodle8/Galacticraft#2074]
See https://github.com/MJRLegends/Space-Astronomy-Feedback-/issues/205 for details

Also the mod pack is using Galacticraft 3.0.12.440 and More Planets addon 1.3.0

**Comment by dj3520** [comment]
If Optifine is being used try again with PlayerAPI from Smart Moving. To
fix this with multiplayer also install RenderPlayerAPI.

Next time you might want to check the wiki for things like this.

**Comment by radfast** [comment]
@MJRLegends  I need to know which other "visual" mods the player has installed.  Exact versions please.

Focus on:
Optifine
PlayerAPI
RenderPlayerAPI
SmartMoving
SmartRender

any other "client-only" mods.

**Comment by MJRLegends** [comment]
@radfast Well PlayerAPI and Fastcraft are in the mod pack by default but not sure if @brucethemoose using any client side mods other than them ones.

**Comment by brucethemoose** [comment]
As seen here:

https://github.com/MJRLegends/Space-Astronomy-Feedback-/issues/205

This turned out to be a conflict with DragonAPI. Updating to the latest version (v10d) doesn't seem to fix the issue.

**Comment by brucethemoose** [comment]
Just to rule out other mods, this happens in a fresh MC install with nothing but the latest builds of Forge, Galacticraft, and DragonAPI:

http://imgur.com/a/qpbTl

**Comment by radfast** [comment]
Thank you @brucethemoose for finding the conflict - that is very useful.

I'll do some testing with DragonAPI and see if we can find a workaround.
Not the first time DragonAPI has messed up Galacticraft, either... :(

**Comment by radfast** [comment]
Please try changing a couple of DragonAPI config options, see if that fixes it.

The ones I'm looking at are:

"Custom/Donator Renders" - try setting that to false
"Disable Renders For Debug" - try setting that to true

**Comment by radfast** [comment]
OK those config settings don't help.

But good news, I found a workaround. :)  This will be in next GC build 443.

Reika's approach to this is not community minded, he breaks all other mods only to give his mods the possibility of showing donator skins.

TL;DR: If you donated to Reika's mods, you're not going to see your special donator skin now, if Galacticraft is installed.  Tough luck.  Reika should use a system which does not break other mods.

**Comment by MJRLegends** [comment]
Ok haha @radfast

**Comment by brucethemoose** [comment]
#Drama

Ya, was about to post that the config changes didn't work for me either.

Sounds good to me, as long as the workaround doesn't break any rendering in Reika's other mods.

**Comment by radfast** [comment]
> as long as the workaround doesn't break any rendering in Reika's other mods.

I can't guarantee that, I don't know all his mods.  But it appears that his breaking system is used only to show donator player skins (and custom skins for Reika and his mate Samaki, when he's playing on his own PC - IKYN).  In any event, the real problem here is (1) he is not using standard Forge methods to change the renderer; (2) he is changing the whole Minecraft renderer only to show a different player _skin_, that's not necessary he could just find a way to change the skin; (3) he is not using RenderPlayerAPI for compatibility with other mods.
